WickedEngine/wiMath_BindLua.h

```diff
@@ -48,19 +48,20 @@ namespace wi::lua
 		int Add(lua_State* L);
 		int Subtract(lua_State* L);
 		int Lerp(lua_State* L);
-		int Rotate(lua_State* L);
```

Comment:
Your merge was incorrect, it has overwritten some of the recent changes and some of my recently added lua bindings are deleted